docs/manual/guides/local-installation/ddev.de.md

```diff
@@ -80,42 +80,58 @@ mkdir -p  ~/Projekte/contao/contao-ddev && cd ~/Projekte/contao/contao-ddev
 DDEV-Konfiguration anlegen mit:
 
 ```shell
-ddev config
+ddev config --project-type=php --docroot=public --webserver-type=apache-fpm --php-version=8.2 --create-docroot
 ```
 
-DDEV-Einstellungen vornehmen, als __Project Type__ auf jeden fall `php` auswählen. Die __Docroot Location__ erstmal leer lassen, da es bei Neuinstallationen noch keinen `public` Ordner gibt und DDEV dann nicht starten kann.
+Contao 5.3 installieren:
 
 ```shell
-ddev start
+ddev composer create contao/managed-edition:5.3
 ```
 
-Zur Installation via Konsole ist es am einfachsten, sich via SSH mit dem Container zu verbinden.
+Nach der Installation müssen die Datenbankzugangsdaten in die `.env` eingetragen werden. In diesem Zug richten wir auch direkt Mailpit ein.
 
-```shell
-ddev ssh
+```env
+APP_ENV=prod
+DATABASE_URL="mysql://db:db@db:3306/db"
+MAILER_DSN="smtp://localhost:1025?encryption=&auth_mode="
```

Comment:
Regarding your specification:
MAILER_DSN=“smtp://localhost:1025?encryption=&auth_mode=” wouldn't MAILER_DSN=“smtp://localhost:1025” with regard to “mailpit” be sufficient? Otherwise you should perhaps describe this in more detail.

Reply:
Other users have recommended this working entry:
`MAILER_DSN=smtp://null:null@localhost:1025`
According to the [ddev documentation](https://ddev.readthedocs.io/en/stable/users/usage/developer-tools/#email-capture-and-review-mailpit).

Reply:
All of this is likely equivalent to `MAILER_DSN=smtp://localhost:1025`.

@fkaminski just test it and report back here

Reply:
`MAILER_DSN="smtp://localhost:1025?encryption=&auth_mode="` works, i just tested.

Reply:
@avhulst the question is whether `MAILER_DSN=smtp://localhost:1025` works too.

Reply:
MAILER_DSN="smtp://localhost:1025" works fine too

Reply:
The quotes are technically unnecessary too

Reply:
MAILER_DSN is updated